src/components/loader/loader.component.js

```diff
@@ -1,9 +1,15 @@
 import React from "react";
 import PropTypes from "prop-types";
+import styledSystemPropTypes from "@styled-system/prop-types";
 import tagComponent from "../../utils/helpers/tags";
 import OptionsHelper from "../../utils/helpers/options-helper";
 import StyledLoader from "./loader.style";
 import StyledLoaderSquare from "./loader-square.style";
+import { filterStyledSystemMarginProps } from "../../style/utils";
+
+const marginPropTypes = filterStyledSystemMarginProps(
+  styledSystemPropTypes.space
+);
 
 const Loader = (props) => {
```

Comment:
We should filter out margin props and apply them only to StyledLoader, now they are passed also to every StyledLoaderSquare.
`const marginProps = filterStyledSystemMarginProps(props);`

Reply:
good point